hack/jenkins/test-history/gen_json.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates a JSON file containing test history for the last day.
+
+Writes the JSON out to tests.json.
+"""
+
+from __future__ import print_function
+
+import json
+import os
+import re
+import subprocess
+import sys
+import time
+import urllib2
+import xml.etree.ElementTree as ET
```

Comment:
Please import modules only (not classes)

https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports